tests/unit/sagemaker/model/test_deploy.py

```diff
@@ -67,11 +67,15 @@
 
 SHAP_BASELINE = '1,2,3,"good product"'
 CSV_MIME_TYPE = "text/csv"
+BUCKET_NAME = "mybucket"
 
 
 @pytest.fixture
 def sagemaker_session():
-    session = Mock()
+    session = Mock(
```

Comment:
nit: can probably get this fixture in common place like conftest to be reused accross and not requiring changes in multiple places?

Reply:
Yeah agreed, I wanted to also, will come back to these if I have time

Reply:
Will tackle later in a separate PR, time permitting